apisix/cli/etcd.lua

```diff
@@ -191,10 +191,21 @@ function _M.init(env, args)
         local version_url = host .. "/version"
         local errmsg
 
-        local res, err = request(version_url, yaml_conf)
-        -- In case of failure, request returns nil followed by an error message.
-        -- Else the first return value is the response body
-        -- and followed by the response status code.
+        local res, err
+        local retry_time = 0
+        while retry_time < 2 do
+            res, err = request(version_url, yaml_conf)
+            -- In case of failure, request returns nil followed by an error message.
+            -- Else the first return value is the response body
+            -- and followed by the response status code.
+            if res then
+                break
+            end
+            retry_time = retry_time + 1
+            print(str_format("Warning! Request etcd endpoint \'%s\' error, %s, retry_time=%s",
+                             version_url, err, retry_time))
```

Comment:
Need a test in https://github.com/apache/apisix/blob/master/t/cli/test_etcd.sh

Reply:
Ping @yxudong

Reply:
> Ping @yxudong

Get it, I‘ll do it soon.